prototype/water-extent-map.ipynb

```diff
@@ -0,0 +1,242 @@
+    "\n",
+    "[S1A_IW_GRDH_1SDV_20210413T235641_20210413T235706_037439_0469D0_3F2B](https://search.asf.alaska.edu/#/?zoom=6.964&center=-90.509,40.296&resultsLoaded=true&granule=S1A_IW_GRDH_1SDV_20210413T235641_20210413T235706_037439_0469D0_3F2B-METADATA_GRD_HD&searchType=List%20Search&searchList=S1A_IW_GRDH_1SDV_20210413T235641_20210413T235706_037439_0469D0_3F2B)\n",
+    "\n",
+    "### 1.1 Connect to HyP3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import shutil\n",
+    "import hyp3_sdk as sdk\n",
+    "\n",
+    "hyp3 = sdk.HyP3(prompt=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.2  Submit an On Demand request for the input RTC product\n",
+    "\n",
+    "Currently, the RTC options provided to `hyp3.submit_rtc_job` as specified are expected/required by `make_water_map` and are not reccomended to be changed, except for:\n",
+    "\n",
+    "* `speckle_filter` -- Apply an Enhanced Lee speckle filter. \n",
+    "\n",
+    "* `dem_matching` -- Coregisters SAR data to the DEM, rather than using dead reckoning based on orbit files. See the [RTC product guide](https://hyp3-docs.asf.alaska.edu/guides/rtc_product_guide/#terrain-correction) for more information.\n",
+    "\n",
+    "* `include_dem` -- include a GeoTIF of the Copernicus DEM pixel aligned to the RTC products. This GeoTIFF isn't used by `make_water_map` but was used to create the RTC product and the HAND data\n",
+    "\n",
+    "* `include_rgb` -- include a false-color RGB decomposition for dual-pol granules. This GeoTIFF isn't used by `make_water_map` but may help you [interpret the RTC image](https://github.com/ASFHyP3/hyp3-lib/blob/develop/docs/rgb_decomposition.md#interpreting-the-images) and evalutate the water extent map\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "granule = 'S1A_IW_GRDH_1SDV_20210413T235641_20210413T235706_037439_0469D0_3F2B'\n",
+    "\n",
+    "job = hyp3.submit_rtc_job(\n",
+    "    granule, name='water-extent-example',\n",
+    "    radiometry='gamma0', scale='power', resolution=30, dem_name='copernicus',\n",
+    "    include_dem=True, include_rgb=True,\n",
+    "    speckle_filter=True, dem_matching=False\n",
+    ")\n",
+    "print(job)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.3 Wait for the RTC product to be generated\n",
+    "\n",
+    "This will take $\\approx$ 25 minutes to complete."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = hyp3.watch(job)"
```

Comment:
`ImportError: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html`

Reply:
Resolved with `conda install ipywidgets` and restarting the kernel